### PR TITLE
Fix job scheduler bug for dependent expiry date job for AB#15242

### PR DESCRIPTION
**Pull request by BrianMaki**
# Fixes [AB#15242](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/15242)

## Description

- Register job in startup via schedule helper

Screenshot 2023-03-27 at 1 22 09 PM

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## UI Changes

No

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

**Review by BronzBierd — APPROVED** [review]

File: Apps/JobScheduler/src/appsettings.json [hunk 1]
```diff
@@ -105,7 +105,7 @@
     },
     "DependentExpiryDate": {
         "Id": "Dependent Expiry Date",
-        "Schedule": "0 */1 * * *",
+        "Schedule": "0 */1 * * * ",
```
Comment (MikeLyttle) [inline Apps/JobScheduler/src/appsettings.json:108]: Why is this change needed?

**Review by MikeLyttle — APPROVED** [review]

**Review by MikeLyttle — APPROVED** [review]

**Review by BronzBierd — APPROVED** [review]
